Get-NetView.psm1

```diff
@@ -2359,6 +2354,10 @@ function Counters {
     }
     $instancesToQuery | Out-File -FilePath $in -Encoding ascii -Width $columns
 
+    if ($SkipCounters) {
```

Comment:
Consider a conditional around the execution below instead of an early exit.  SImplistically to be more consistent with most of the code in this file.